airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.module.scss

```diff
@@ -0,0 +1,91 @@
+@use "scss/colors";
+
+.jobListContainer {
+  padding: 25px 25px 22px;
+  background-color: colors.$grey-50;
+}
+
+.jobListTitle {
+  display: flex;
+  justify-content: space-between;
+}
+
+.jobListForm {
+  width: 100%;
+}
+
+.emptyListContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  > img {
```

Comment:
:art: We have that `img` element under our control as well (i.e. render it in JSX), so it would be preferable to give the img itself just a classname instead of selecting `> img` here.